lib/pageflow/global_config_api.rb

```diff
@@ -48,10 +48,11 @@ def config_for(target)
         c.enable_features(target.enabled_feature_names)
       end
       if target.respond_to?(:type_name)
-        ConfigDelegator.new(config, target.type_name)
-      else
-        config
+        config = ConfigDelegator.new(config, target.type_name)
+        config.configure_blocks.each { |block| block.call(config) }
```

Comment:
This causes entry type configure blocks to be executed after the `after_configure` blocks, which would break integrations like the ones we've built to register file types etc for page types. I think the entry type specific configure blocks need be evaluated during `build_config` at the same time when `enable_feature` is called above.

Let's cover this case with a spec.